src/ansys_sphinx_theme/theme/ansys_sphinx_theme/theme.conf

```
@@ -17,4 +17,5 @@ footer_end = theme-version.html
 pygment_light_style = friendly
 pygment_dark_style = monokai
 cheatsheet =
-ansys_sphinx_theme_autoapi =
\ No newline at end of file
+ansys_sphinx_theme_autoapi =
+logo =
```

Shouldn't this use the default value?

```suggestion
logo = pyansys
```

---

> Shouldn't this use the default value?

i am adding the default value by the code, and wanted to keep the nature of dict from pydata sphinx theme, so we dont need to keep the value here